docs/source/pages/documentation.md

```diff
@@ -1,18 +1,170 @@
 # Full Documentation
 
-Datashuttle is a work in progress as has not been officially released. It is not ready for use
-as documented, please await first official release.
 
-DataShuttle helps to manage and transfer a project with many "local" machines all connected to a "central" machine.
-DataShuttle has functions to help:
-* Generate BIDS-formatted folder structures
-* Transfer data between central and local machines
+DataShuttle is a tool to streamline the management of neuroscience projects, allowing simple and convenient standardisation of data structures.
+
+DataShuttle's goal is to alleviate the burden researchers face in adhering to standardized file and folder specifications during the execution of intricate and demanding experimental projects. It will:
+
+- Eliminate the need to manually integrate datasets collected across different machines (e.g. behaviour and electrophysiology acquisition machines).
+- Allow convenient transfer of data between machines. This may be between a central project storange and analysis machine (e.g. ''*I want to transfer subjects 1-5, sessions 5 and 10, behavioural data only to my laptop*.'')
```

Comment:
typo
```suggestion
- Allow convenient transfer of data between machines. This may be between a central project storage and analysis machine (e.g. ''*I want to transfer subjects 1-5, sessions 5 and 10, behavioural data only to my laptop*.'')
```